### PR TITLE
Fixing redis config for 14.04. PECL no loner works, so using apt now.

### PR DESCRIPTION
**Pull request by sk33lz**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -56,17 +56,20 @@
   include php::apache
 
   include php::cli
-  # php module wants to use apt as the provider but the package isn't available on ubuntu 12.04.
-  class { 'php::extension::redis':
-    provider => 'pecl',
```
Comment (lliss) [inline manifests/init.pp:61]: You will want to keep this class and just change the provider to apt and the package to php5-redis.

File: manifests/init.pp [hunk 2]
```diff
@@ -56,17 +56,20 @@
...
-    package  => 'redis',
-  }->
-
-  php::config { 'redis_conf':
```
Comment (lliss) [inline manifests/init.pp:65]: We'll also want to keep the config.

**Comment by sk33lz** [comment]
Fixed and squashed.

**Comment by lliss** [comment]
+1 tested.

**Pull request merged**